source/docs/software/pathplanning/pathweaver/creating-pathweaver-project.rst

```diff
@@ -34,9 +34,9 @@ To create a PathWeaver project, click on "Create project" and then fill out the
 
 **Export Unit:** The units to be used when exporting the paths and waypoints. If you are planning to use WPILib Trajectories, then you should choose :guilabel:`Always Meters`. Choosing :guilabel:`Same as Project` will export in the same units as :guilabel:`Length Unit` above.
 
-**Max Velocity:** The max speed of the robot for trajectory tracking. This does not need to be the maximum attainable speed of the robot, but just the max speed that you want to run trajectories at.
+**Max Velocity:** The max speed of the robot for trajectory tracking.  The kitbot runs at ~10 :math:`ft/sec` which is ~3 :math:`m/sec`.
```

Comment:
Why remove the part about it being for the trajectory not the max speed of the robot. Since you normally want some headroom, I'd suggest 80% of robot max speed.

Reply:
@calcmogul ?  As I was understanding what he was saying this value should be the maximum and teams should add constraints later.  Either way all of this is kindof bogus until we decide to go all in on either constraints in the program or constraints on RIO (and thus no accurate preview)

Reply:
> As I was understanding what he was saying this value should be the maximum and teams should add constraints later.

The prevailing wisdom for trajectory optimization solvers is to not provide a WAG for an initial velocity profile. You provide an initial guess at a path, then the rest is by the solver via the user-provided dynamics constraints (in general, dx/dt = f(x) + Bu).

WPILib splines use an initial velocity profile to provide a starting point for the time parameterization. Relying on the profile behavior vs using constraints is a trade off between commonly accepted general practice in motion planning and the quickest thing that gets a robot to move. While I see value in the latter since you don't need a model to use with the dynamics constraints, you need a model anyway for good feedforward. DifferentialDriveVoltageConstraint takes a feedforward object. The feedforward gains can be increased to provide headroom.

> Either way all of this is kindof bogus until we decide to go all in on either constraints in the program or constraints on RIO (and thus no accurate preview)

PathWeaver currently only provides a path preview, and constraints don't change the path shape. A time domain preview needs constraints though.